services/frontend-service/src/ui/components/dialog/ConfirmationDialog.tsx

```diff
@@ -73,7 +73,19 @@ export const PlainDialog: React.FC<PlainDialogProps> = (props) => {
                 onClose();
             }
         });
-    }, [onClose]);
+        document.addEventListener('click', (event) => {
+            if (open) {
+                if (event.target instanceof HTMLElement) {
+                    // eslint-disable-next-line no-type-assertion/no-type-assertion
+                    const target = event.target as HTMLElement;
```

Comment:
Is there no way to get around the `as HTMLElement`? :(

Reply:
I am not aware of one.

`event.target`s type is very generic.

Reply:
Having the condition in the line 78 doesn't remove the need for the cast?

Reply:
You are right, it works :smiley: 
I simplified it